### PR TITLE
Add terms, relations and definitions to expand Dataset and DataCatalog

### PR DESCRIPTION
**Pull request by klngwll**

### CODE REVIEW

File: source/vocab/platform.ttl [hunk 1]
```diff
@@ -229,18 +230,36 @@
...
 :hasDataset a owl:ObjectProperty;
     rdfs:label "har dataset"@sv;
     rdfs:domain :DataCatalog;
     rdfs:range :Dataset;
     owl:equivalentProperty sdo:dataset, dcat:dataset;
+    owl:inverseOf :includedInDataCatalog ;
     rdfs:subPropertyOf dc:hasPart .
 
+:includedInDataCatalog a owl:ObjectProperty;
+    rdfs:label "del av katalog"@sv;
```
Comment (olovy) [inline source/vocab/platform.ttl:256]: "ingår i katalog" ?

File: source/vocab/platform.ttl [hunk 2]
```diff
@@ -229,18 +230,36 @@
...
 
 :hasDataset a owl:ObjectProperty;
     rdfs:label "har dataset"@sv;
     rdfs:domain :DataCatalog;
     rdfs:range :Dataset;
     owl:equivalentProperty sdo:dataset, dcat:dataset;
+    owl:inverseOf :includedInDataCatalog ;
     rdfs:subPropertyOf dc:hasPart .
 
+:includedInDataCatalog a owl:ObjectProperty;
```
Comment (olovy) [inline source/vocab/platform.ttl:255]: `inDataCatalog` is more consistent with out other properties? 
Maybe `includedInDataCatalog`is more clear (and aligned with sdo)

File: source/vocab/things.ttl [hunk 3]
```diff
@@ -237,10 +237,11 @@
...
 :Dataset a owl:Class;
-    rdfs:label "Dataset"@sv;
-    skos:definition "Data kodad i en definierad struktur. Inkluderar bland annat numeriska data, miljödata. Används av applikationsprogramvara för att beräkna medelvärden, korrelationer, eller för att producera modeller etc. Visas normalt inte i sin råa form."@sv;
-    rdfs:subClassOf :Work;
-    owl:equivalentClass dctype:Dataset, bf2:Dataset, sdo:Dataset .
+    # NOTE: See :Dataset in platform.ttl for more detailed descriptions and vocabulary correlations regarding the notion of Datasets.
+    # rdfs:label "Dataset"@sv;
+    # skos:definition "Data kodad i en definierad struktur. Inkluderar bland annat numeriska data, miljödata. Används av applikationsprogramvara för att beräkna medelvärden, korrelationer, eller för att producera modeller etc. Visas normalt inte i sin råa form."@sv;
+    # rdfs:subClassOf :Work;
+    owl:equivalentClass bf2:Dataset .
```
Comment (olovy) [inline source/vocab/things.ttl:244]: `owl:equivalentClass bf2:Dataset` defined both here and in platform.ttl?

Comment (klngwll) [reply source/vocab/things.ttl:244]: Becomes the same statement. I used it as a security measure ;)

File: source/vocab/platform.ttl (hunk 2 above)
Comment (klngwll) [reply source/vocab/platform.ttl]: Let's go for consistency.

**Review by olovy — APPROVED** [review]
LGTM!

**Pull request merged**